OWNERS

```diff
@@ -3,9 +3,9 @@
 approvers:
 - kubernetes-csi-approvers
 - bswartz
+- sunnylovestiramisu
 
 reviewers:
 - kubernetes-csi-reviewers
 - bswartz
-- sunnylovestiramisu
```

Comment:
I am already on the kubernetes-csi-reviewers list, so removing this will not impact reviewing.